docs/other/guides/how-to-create-scaffolding-templates.md

```diff
@@ -0,0 +1,45 @@
+---
+id: creatingScaffoldingTemplates
+title: How to create Scaffolding Templates
+sidebar_label: Creating Scaffolding Templates
+---
+
+### Introduction
+As discussed in [Scaffolding](/docs/get-started/scaffolding), it is possible to save yourself a lot of time by using one of the default Scaffolding Templates.
```

Comment:
`it is possible to save yourself a lot of time` -> `you can save a lot of time`.
`by using one of the default Scaffolding Templates` -> `by using scaffolding templates`

Reply:
done